main/pom.xml

```diff
@@ -81,7 +81,6 @@
             <groupId>org.hamcrest</groupId>
             <artifactId>hamcrest</artifactId>
             <version>2.2</version>
-            <scope>test</scope>
```

Comment:
@jsolomon8080 I'll fix this here as it should be listed as optional now.  That way others don't pick it up.